recipes/r-metama/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = '3.1.2' %}
+
+package:
+  name: r-metama
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: metaMA_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/metaMA_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/metaMA/metaMA_{{ version }}.tar.gz
+  sha256: 01851d9632cdf1988d6a43ba44500e51477b9b25d5df1e1da488cd800df85ed6
+
+build:
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-smvar
+    - bioconductor-limma
+
+  run:
+    - r-base
+    - r-smvar
+    - bioconductor-limma
+
+test:
+  commands:
+    - $R -e "library('metaMA')"      
+
+about:
+  home: https://CRAN.R-project.org/package=metaMA
+  license: GPL
+  summary: Combines either p-values or modified effect sizes from different studies to find differentially
+    expressed genes
+  license_family: GPL
```

Comment:
Can you please add:
`license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'  # [unix]`

Reply:
With this line, the Travis CI build failed :
https://travis-ci.org/bioconda/bioconda-recipes/jobs/313376197